rune/libenclave/internal/runtime/pal/pal_linux.go

```diff
@@ -48,6 +86,10 @@ func (pal *enclaveRuntimePal) Kill(pid int, sig int) error {
 
 func (pal *enclaveRuntimePal) Destroy() error {
 	api := &enclaveRuntimePalApiV1{}
+
+	if isEncCache == false && hasEpm == true {
```

Comment:
If pal.ID is set to an invalid epm ID earlier, using `pal.ID != InvalidEpmID` is good enough.

Reply:
Good idea , I will change. Thanks